exec/java-exec/src/main/java/org/apache/drill/exec/vector/complex/AbstractMapVector.java

```diff
@@ -189,8 +189,8 @@ protected void putVector(String name, ValueVector vector) {
         Preconditions.checkNotNull(vector, "vector cannot be null")
     );
     if (old != null && old != vector) {
-      logger.debug("Field [%s] mutated from [%s] to [%s]", name, old.getClass().getSimpleName(),
-          vector.getClass().getSimpleName());
+      logger.debug("Field [{}] mutated from [{}] to [{}]", name, old.getClass().getSimpleName(),
```

Comment:
http://www.slf4j.org/api/org/slf4j/helpers/MessageFormatter.htmlDoesn't the number of brace replacement patterns correspond to the number of arguments?